Makefile

```diff
@@ -233,7 +230,7 @@ check-python-packages:
 	@echo ""
 	@echo "================== CHECK PYTHON PACKAGES ===================="
 	@echo ""
-	test -f $(VIRTUALENV_COMPONENTS_DIR)/bin/activate || virtualenv --python=$(PYTHON_VERSION) $(VIRTUALENV_COMPONENTS_DIR) --no-download --system-site-packages
+	test -f $(VIRTUALENV_COMPONENTS_DIR)/bin/activate || $(PYTHON_VERSION) -m venv $(VIRTUALENV_COMPONENTS_DIR) --system-site-packages
```

Comment:
I like the transition to native py3 `venv`, however we stick to `virtualenv` tool and its behavior in many other places, including packaging, as well as other CIs using this st2 Makefile. I suspect that this might provide problems and looks like there were already issues related to compatibility with the previous tool.

Can we keep the old `virtualenv` approach for this PR and do `venv` migration in another isolated PR?
So this PR would do 1 thing, and do it well: removing python2 only (as advertized).

Reply:
Done. :+1: